packages/fetchye/__tests__/index.spec.js

```diff
@@ -20,9 +20,25 @@ describe('index', () => {
   it('should return public methods', () => {
     expect(Object.keys(fetchye).sort()).toMatchInlineSnapshot(`
       Array [
+        "ACTION_NAMESPACE",
+        "CLEAR_ERROR",
+        "DELETE_DATA",
+        "ERROR",
+        "FetchyeContext",
         "FetchyeProvider",
+        "IS_LOADING",
+        "SET_DATA",
+        "SimpleCache",
+        "clearErrorsAction",
+        "defaultEqualityChecker",
+        "defaultFetcher",
+        "deleteAction",
+        "errorAction",
+        "loadingAction",
         "makeServerFetchye",
+        "setAction",
         "useFetchye",
+        "useSubscription",
```

Comment:
@JAdshead regarding the structure of the API, do you think methods like `useSubscription`, the cache actions etc should not be part of the public API to use Fetchye but more like configurations / internal APIs that are used to create cacheProviders / adapters? 

Only concern is if in the future there is a `useSubscription` hook to call GraphQL subscriptions for example that we want to add as a public API, that name is already taken by a private method?